File: Add Code Here/C#/Calculator.cs

using System;
using System.Text;
using System.Threading.Tasks;

namespace calculator_c_sharp
{
	class Program
	{
		static void Main(string[] args)
		{
			string value;
			do
			{
				int res;
				Console.Write("Enter first number:");
				int num1 = Convert.ToInt32(Console.ReadLine());
				Console.Write("Enter second number:");
				int num2 = Convert.ToInt32(Console.ReadLine());
				Console.Write("Enter symbol(/,+,-,*):");
				string symbol = Console.ReadLine();

				switch (symbol)
				{
					case "+":
						res = num1 + num2;
						Console.WriteLine("Addition:" + res);
						break;
					case "-":
						res = num1 - num2;
						Console.WriteLine("Subtraction:" + res);
						break;
					case "*":
						res = num1 * num2;
						Console.WriteLine("Multiplication:" + res);
						break;
					case "/":
						res = num1 / num2;
						Console.WriteLine("Division:" + res);
						break;
					default:
						Console.WriteLine("Wrong input");
						break;
				}
				Console.ReadLine();
				Console.Write("Do you want to continue(y/n):");
				value = Console.ReadLine();
			}
			while (value=="y" || value=="Y");
		}
	}

}

File: Add Code Here/PYTHON/calculator.py
# Function to add two numbers
def add(x, y):
    return x + y

# Function to subtract two numbers
def subtract(x, y):
    return x - y

# Function to multiply two numbers
def multiply(x, y):
    return x * y

# Function to divide two numbers
def divide(x, y):
    # Check if the denominator is not zero
    if y != 0:
        return x / y
    else:
        return "Error: Division by zero is not allowed."

# Main function
def main():
    print("Calculator Menu:")
    print("1. Addition (+)")
    print("2. Subtraction (-)")
    print("3. Multiplication (*)")
    print("4. Division (/)")

    # Get operator choice from the user
    choice = input("Enter operator (1/2/3/4): ")

    # Get two numbers from the user
    num1 = float(input("Enter first number: "))
    num2 = float(input("Enter second number: "))

    # Perform calculation based on the operator choice
    if choice == '1':
        print("Result: ", add(num1, num2))
    elif choice == '2':
        print("Result: ", subtract(num1, num2))
    elif choice == '3':
        print("Result: ", multiply(num1, num2))
    elif choice == '4':
        print("Result: ", divide(num1, num2))
    else:
        print("Invalid operator choice.")

# Run the calculator program
if __name__ == "__main__":
    main()
